### ISSUE TITLE
Chrome console not outputting error thrown in subscribe callback

### ISSUE THREAD

**Issue by BuildingJarl** [aurelia/event-aggregator#26]
**I'm submitting a bug report**
- **Library Version:**
  1.0.0

**Please tell us about your environment:**
- **Operating System:**
  OSX 10.12.1
- **Node Version:**
  6.6.0
  
  
- **NPM Version:**
  3.10.3
  
  
- **JSPM OR Webpack AND Version**
  JSPM 0.16.47
  
  
- **Browser:**
  Chrome 54.0.2840.71 (64-bit)
- **Language:**
  ESNext

**Current behavior:**
If an error is thrown in a subscribe callback e.g.

```
eventAggregator.subscribe('some_event', ()=>{
    //some error here
})
```

error is not output in chrome console. If a breakpoint is set, then error can be seen being caught by catch statement. Please see image below

![screen shot 2016-10-27 at 15 31 52](https://cloud.githubusercontent.com/assets/5493114/19771480/f8118b68-9c5a-11e6-82ad-cfdb49d8d635.png)

**Expected/desired behavior:**

[Runnable gist](https://gist.run/?id=2c12f00aa33ca62115e8c9b0d89ed454)

check app.js
- **What is the expected behavior?**
- **What is the motivation / use case for changing the behavior?**

**Comment by StrahilKazlachev** [comment]
Either inappropriate log level(none - 0) or bug in aurelia-logging.

**Comment by BuildingJarl** [comment]
@StrahilKazlachev  just added a runnable gist

**Comment by StrahilKazlachev** [comment]
It's a bug, fixed aurelia/logging#26, but not released at the moment.

**Comment by EisenbergEffect** [comment]
Closing this as dupe. Fix coming.
